data/json/items/armor/helmets.json

```diff
@@ -20,6 +20,57 @@
     "techniques": [ "WBLOCK_1" ],
     "armor": [ { "encumbrance_modifiers": [ "IMBALANCED" ], "coverage": 55, "covers": [ "head" ] } ]
   },
+  {
+    "id": "stockpot_helmet",
+    "repairs_like": "mc_helm_close",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "pot great helm" },
+    "description": "An enclosed helmet made from a stock pot with a 'T' crudely carved into the front and a blanket stuffed inside to cushion it.  There are strings tied to the handles to help fasten it to your body, but it's extremely limiting and smothering.",
+    "weight": "3580 g",
+    "volume": "13 L",
+    "price": 4500,
+    "price_postapoc": 50,
+    "bashing": 10,
+    "material": [ "mc_steel", "aluminum" ],
+    "symbol": "[",
+    "looks_like": "helmet_plate",
+    "color": "light_gray",
+    "warmth": 70,
+    "material_thickness": 1,
+    "qualities": [ [ "COOK", 2 ], [ "BOIL", 1 ] ],
```

Comment:
There are holes in it now, I don't think it's going to be as good for cooking anymore

Reply:
yeah, it went to boiling level 1 instead since saucepans have boiling 1 and the cuts don't go all the way up the pot. So it's no longer super suitable for boiling (stock pot has 2 boil) but it can still do it a little bit